pkg/wallet/query.go

```diff
@@ -98,7 +98,7 @@ func (q *Query) PerformQuery(credentials map[string]json.RawMessage) ([]*verifia
 
 		credentials, err := q.getCredentials(qType, vcs, param.Query...)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("getCredentials: %w", err)
```

Comment:
what is the reason for prefixing function names here?

Reply:
Added them for more verbose logging during debugging - I can remove these, they're not really necessary.

Reply:
you shouldn't put function names in the message.